src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -263,108 +268,212 @@ export const bankFieldsHaveInput = formData =>
     'view:bankAccount.bankName',
   ]);
 
-/**
- * Creates uiSchema and schema for address widget based on params
- * @param {array} addressOmitions
- * @param {array} order
- * @param {object} fieldLabels
- */
-export function generateAddressSchemas(addressOmitions, order, fieldLabels) {
-  const addressSchemaConfig = addressSchema(fullSchema);
-  const addressUIConfig = omit(addressUI(' '), addressOmitions);
-
-  const locationSchema = {
-    addressUI: {
-      ...addressUIConfig,
-      'ui:order': order,
-    },
-    addressSchema: {
-      ...addressSchemaConfig,
-      properties: {
-        ...omit(addressSchemaConfig.properties, addressOmitions),
-      },
-    },
-  };
-
-  if (!addressOmitions.includes('country')) {
-    locationSchema.addressUI.country = {
-      'ui:title': fieldLabels.country,
-    };
+export const AddressViewField = ({ formData }) => {
```

Comment:
So...git diff kinda sucks for this. Basically I deleted `generateAddressSchemas` and put this in. `AddressViewField` is unchanged.

Reply:
Kind of surprised this component is in `/utils` in the first place 🤷‍♂️

Reply:
Yeah, the `content/` directory has a bunch of files for each page. It seemed like this should stay in `content/`, but I didn't have a corresponding page to put it in, and it seemed wrong to make `util` depend on a `content/pageName.js` file.